frontend/public/models/yaml-templates.ts

```diff
@@ -7,6 +7,10 @@ import { YAMLTemplate } from '@console/plugin-sdk';
 /**
  * Sample YAML manifests for some of the statically-defined Kubernetes models.
  */
+
+const containerImageSrc = 'registry.redhat.io/rhel8/support-tools';
+const containerCmd = '[ "/bin/bash", "-c", "sleep infinity" ]';
```

Comment:
I'd put "sample" in the variable name to make it clear these are just samples.

```suggestion
const sampleContainerImage = 'registry.redhat.io/rhel8/support-tools';
const sampleContainerCmd = '[ "/bin/bash", "-c", "sleep infinity" ]';
```

Reply:
> I'd put "sample" in the variable name to make it clear these are just samples.

@spadgett Addressed, thanks